source/change-streams/change-streams.rst

```diff
@@ -749,14 +749,14 @@ It was decided to remove this example from the specification for the following r
 - There is something to be said for an API that allows cooperation by default. The model in which a call to next only blocks until any response is returned (even an empty batch), allows for interruption and cooperation (e.g. interaction with other event loops).
 
 ----------------------------------------
-Why is a whitelist of error codes preferable to a blacklist?
+Why is an allow list of error codes preferable to a deny list?
```

Comment:
I realize the original file had the same error, but this would be a good chance to ensure the dash lines have the same width as the title.

Reply:
Fair point. Fixed.